doc/source/database/cratedb.rst

```diff
@@ -0,0 +1,542 @@
+.. include:: ../_resources.rst
+
+.. _database-cratedb:
+
+#######
+CrateDB
+#######
+
+
+*****
+About
+*****
+
+`CrateDB`_ is a distributed and scalable SQL database for storing and analyzing massive
+amounts of data in near real-time, even with complex queries. It is PostgreSQL-compatible,
+and based on `Lucene`_.
+
+
+*******
+Details
+*******
+
+This section summarizes CrateDB's data model and query interface.
+
+Data model
+==========
+
+As a time-series/document/OLAP/RDBMS database with an SQL interface, CrateDB stores records
+into tables. Tables are grouped into schemas, which is equivalent to the concept of hosting
+multiple databases on the same server instance.
```

Comment:
@proddata mentioned that, at least from a PostgreSQL perspective, databases are more like catalogs. Thanks.